macros/edr/dbt_artifacts/upload_dbt_exposures.sql

```diff
@@ -1,8 +1,15 @@
 {%- macro upload_dbt_exposures() -%}
+    {% set identifier = 'dbt_exposures' %}
+    {% if results and elementary.get_result_node('model.elementary.%s' % identifier) %}
```

Comment:
cosmetics - I'd get the unique id from the node of dbt_exposures

Reply:
Well, this is the same logic, if it failed why would it succeed on the second attempt?

> I'd get the unique id from the node of dbt_exposures

Sorry, could you please clarify what do you mean? This is the `unique_id`.

<img width="509" alt="image" src="https://user-images.githubusercontent.com/30181361/196045111-04fe8ced-1484-434b-a7e3-82ac459c1334.png">

Reply:
I know but if tomorrow dbt will change the format this will longer be valid so I'd rather find the node and get the unique id from the node (you rely on their format)

Reply:
I can change the signature to be `get_result_node(package_name, identifier)`, think that's more appropriate? Might be a better API in general.

However I'd say that I don't think changing the unique id format is something they're likely to do out of the blue without prior notice.